Public/Src/Engine/Processes/SandboxedProcess.MacKext.cs

```diff
@@ -165,11 +207,8 @@ private async Task OnProcessStartedAsync()
 
             if (MeasureCpuTime)
             {
-                // Allow read access for /usr/bin/time
-                ProcessInfo.FileAccessManifest.AddPath(
-                    AbsolutePath.Create(PathTable, TimeUtil),
-                    mask: FileAccessPolicy.MaskNothing,
-                    values: FileAccessPolicy.AllowReadAlways);
+                // start the timer now
+                m_perfTimer.Change(TimeSpan.FromSeconds(0), PerfProbeInternal);

```

Comment:
>m_perfTimer.Change(TimeSpan.FromSeconds(0), PerfProbeInternal); [](start = 16, length = 63)

A bit less intuitive but oftentimes the safer pattern is to only ever set the due time and not the period on the timer. Then when it fires you call Change() with a new due time in your timer callback. That prevents the timer from firing again while its still in the callback which can cause some unintended concurrency issues in the callback

Reply:
will change